server/src/service-layer/response-models/BookingResponse.ts

```diff
@@ -17,3 +17,11 @@ export interface UsersByDateRangeResponse {
   data?: Array<{ date: Timestamp; users: CombinedUserData[] }>
   error?: string
 }
+
+/**
+ * Represents the response structure for fetching user ids by date range.
+ */
+export interface UIdssByDateRangeResponse {
```

Comment:
Just wanted to clarify if this was okay to do or was I meant to use another response?

Reply:
yea should be fine